docs/photoniq/prerendering/prerendering-settings.md

```diff
@@ -0,0 +1,97 @@
+---
+sidebar_position: 60
+title: Prerendering Settings
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+You have several options for viewing and managing Dynamic Prerendering settings.
+
+- **Advanced Settings tab** - You can view settings for an origin in Advanced Settings tab in the web console.
+- **REST API** - Macrometa provides several API calls that allow you to view and manage origin settings.
+
+:::note
+Each setting is explained in detail in the Dynamic Prerendering web console. We strongly recommend that you review the setting and consult with Macrometa Support before making any changes.
+:::
+
+## View Settings
+
+<Tabs groupId="operating-systems">
+<TabItem value="console" label="Web Console">
+
+Follow these instructions to view prerendering activity in the web console.
+
+1. Log in to your Dynamic Prerendering dashboard.
+2. Click **Advanced Setting**.
+3. Select the origin for which you want to view settings.
+4. Scroll down to the setting that you want to view
+
+</TabItem>
+<TabItem value="api" label="REST API">
+
+Use our interactive API Reference with code generation in 18 programming languages to:
+
+- [Get prerender settings](https://www.macrometa.com/docs/apiPrerendering#/paths/api-prerender-v1-origins-origin--settings/get).
+- [Get all origin settings](https://www.macrometa.com/docs/apiPrerendering#/paths/api-prerender-v1-origins/get)
+- [Get settings for an origin](https://www.macrometa.com/docs/apiPrerendering#/paths/api-prerender-v1-origins/get)
```

Comment:
Here, you are referencing the wrong endpoint. This is the correct one:
`https://www.macrometa.com/docs/apiPrerendering#/paths/api-prerender-v1-origins-origin/get`
The label could be `Get origin settings` instead, as we have `Update origin settings` right below.

Reply:
Yes, I took some liberties with the labels, because this difference is really subtle and not easily understandable without the context of the API reference (and maybe not even with):

- Get origins
- Get origin

Reply:
Updated